frontend/src/views/main/project/ProjectProperties.vue

```diff
@@ -0,0 +1,313 @@
+              <v-btn value="datasets" class="py-5 px-4">
+                <span>Datasets</span>
+                <v-icon end class="pb-1 pl-1" :color="toggleObject === 'datasets' ? 'primary' : ''">
+                  stacked_bar_chart
+                </v-icon>
+              </v-btn>
+              <v-btn value="models" class="py-5 px-4">
+                <span>Models</span>
+                <v-icon end class="pb-1 pl-1" :color="toggleObject === 'models' ? 'primary' : ''">
+                  settings_suggest
+                </v-icon>
+              </v-btn>
+            </v-btn-toggle>
+            <div class="flex-1"></div>
+          </v-card-title>
+          <v-card-text>
+            <Datasets v-show="toggleObject === 'datasets'" ref="datasetsComponentRef" :projectId="projectId" :isProjectOwnerOrAdmin="isProjectOwnerOrAdmin"></Datasets>
+            <Models v-show="toggleObject === 'models'" ref="modelsComponentRef" :projectId="projectId" :isProjectOwnerOrAdmin="isProjectOwnerOrAdmin"></Models>
+          </v-card-text>
+        </v-card>
+      </v-col>
+    </v-row>
+
+    <v-dialog persistent max-width="340" v-model="openDeleteDialog">
+      <v-card>
+        <v-card-title>
+          Are you sure you want to delete project?
+        </v-card-title>
+        <v-card-text class="accent--text">
+          All data and files will be lost!
+        </v-card-text>
+        <v-card-actions>
+          <v-spacer></v-spacer>
+          <v-btn color="secondary" text @click="openDeleteDialog = false">Cancel</v-btn>
+          <v-btn color="accent" text @click="deleteProject();">Ok</v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import { getUserFullDisplayName } from '@/utils';
+import Models from '@/views/main/project/Models.vue';
+import Datasets from '@/views/main/project/Datasets.vue';
+import { IUserProfileMinimal } from '@/interfaces';
+import mixpanel from "mixpanel-browser";
+import { useProjectStore } from "@/stores/project";
+import { computed, ref } from 'vue';
+import { $vfm } from 'vue-final-modal';
+import ConfirmModal from '@/views/main/project/modals/ConfirmModal.vue';
+import InlineEditText from '@/components/InlineEditText.vue';
+import { ProjectPostDTO } from '@/generated-sources';
+import { useRouter } from "vue-router/composables";
+
+
+
+interface Props {
+  projectId: number;
+  isProjectOwnerOrAdmin: boolean;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  isProjectOwnerOrAdmin: false
+});
+
+const router = useRouter();
+
+const projectStore = useProjectStore();
+
+const openDeleteDialog = ref(false);
+const datasetsComponentRef = ref<any>(null);
+const modelsComponentRef = ref<any>(null);
+const toggleObject = ref<string>("datasets");
+
+const project = computed(() => useProjectStore().project(props.projectId))
+
+function reloadDataObjects() {
+  datasetsComponentRef.value.loadDatasets();
```

Comment:
I feel that a store would be better to synchronize between components

Reply:
Should I create a single store to store both types of data objects (datasets and models) like `data-object.ts`, or should I create a store for each of them, like `dataset.ts` and `model.ts`?

Reply:
Well we should probably have one `project-artifacts.ts` store that fetch those together when we load the projects. Initially I put all in `test-suite.ts`. However the dataset and model are more global so we should probably extract them.